superset-frontend/src/features/annotations/AnnotationModal.tsx

```diff
@@ -70,7 +71,7 @@ const AnnotationContainer = styled.div`
   }
 
   textarea,
-  input[type='text'] {
+  Input {
```

Comment:
It seems to be intentional at this point? I might be surprised it this actually works :)

Reply:
ooops, used "Aider" here and it went rogue... styles not required anymore...